include/TrustWalletCore/TWTransactionCompiler.h

```diff
@@ -29,7 +29,7 @@ TWData *_Nonnull TWTransactionCompilerBuildInput(enum TWCoinType coinType, TWStr
 /// Obtain pre-signing hashes of a transaction. For each hash its corresponding publickKeyHash follows, so return is always even size.
 /// For chains with one signature only, the result contains 2 elements, but the second may be ignored.
 TW_EXPORT_STATIC_METHOD
-struct TWDataVector *_Nonnull TWTransactionCompilerPreImageHashes(enum TWCoinType coinType, TWData *_Nonnull txInputData);
+TWData *_Nonnull TWTransactionCompilerPreImageHashes(enum TWCoinType coinType, TWData *_Nonnull txInputData);
```

Comment:
it only return one hash now?

Reply:
It will return serialization of `PreSigningOutput` proto, which we put hash, preImage and some error info in it.